CRM/Core/BAO/CustomField.php

```diff
@@ -2256,7 +2256,8 @@ public static function postProcess(
    * @throws \CiviCRM_API3_Exception
    */
   public static function getCustomFieldID($fieldName, $groupTitle = NULL, $fullString = FALSE) {
```

Comment:
Would it make sense to rename `$groupTitle` to `$groupName` at this point to "suggest" what we would prefer it to be?

Reply:
Sounds reasonable, done.